packages/bisect_ppx/bisect_ppx.0.2.2/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "bisect_ppx"
+version: "0.2.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/bisect_ppx"
+dev-repo: "https://github.com/rleonid/bisect_ppx.git"
+bug-reports: "https://github.com/rleonid/bisect_ppx/issues"
+build: [
+  ["sh" "configure"]
+  [make "all"]
+  [make "install"]
```

Comment:
What does your `make install` command do exactly? If it installs a library, you should have the symmetric `ocamlfind remove <foo>` in the `remove` field. And it's also better to have `make install` in a separate `install` field.